tests/scripts/test-ci.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -ex
+
+function push() {
+    arm64_Master_Image=$(docker images --format "{{.Repository}}:{{.Tag}}" | grep "arm64:latest")
+    docker tag "$arm64_Master_Image" quay.io/rook/rook-arm64:latest
+    docker push quay.io/rook/rook-arm64:latest
+    docker tag "$arm64_Master_Image" ghcr.io/rook/rook-arm64:latest
+    docker push ghcr.io/rook/rook-arm64:latest
+
+    arm64_Release_Image=$(docker images --format "{{.Repository}}:{{.Tag}}" | grep -v "arm64:latest" | grep -v "arm64:master" | grep arm64)
+    tag_arm64_Release_Image="$(echo "$arm64_Release_Image" | cut -d ":" -f 2)"
+    docker tag "$arm64_Release_Image" quay.io/rook/rook-arm64:"$tag_arm64_Release_Image"
+    docker push quay.io/rook/rook-arm64:"$tag_arm64_Release_Image"
+    docker tag "$arm64_Release_Image" ghcr.io/rook/rook-arm64:"$tag_arm64_Release_Image"
+    docker push ghcr.io/rook/rook-arm64:"$tag_arm64_Release_Image"
+
+    amd64_Master_Image=$(docker images --format "{{.Repository}}:{{.Tag}}" | grep "amd64:latest")
+    docker tag "$amd64_Master_Image" quay.io/rook/rook-amd64:latest
+    docker push quay.io/rook/rook-amd64:latest
+    docker tag "$amd64_Master_Image" ghcr.io/rook/rook-amd64:latest
+    docker push ghcr.io/rook/rook-amd64:latest
+
+    amd64_Release_Image=$(docker images --format "{{.Repository}}:{{.Tag}}" | grep "amd64" | grep -v "amd64:latest" | grep -v "amd64:master")
+    tag_amd64_Release_Image="$(echo "$amd64_Release_Image" | cut -d ":" -f 2)"
+    docker tag "$amd64_Release_Image" quay.io/rook/rook-amd64:"$tag_amd64_Release_Image"
+    docker push quay.io/rook/rook-amd64:"$tag_amd64_Release_Image"
+    docker tag "$amd64_Release_Image" ghcr.io/rook/rook-amd64:"$tag_amd64_Release_Image"
+    docker push ghcr.io/rook/rook-amd64:"$tag_amd64_Release_Image"
+
+    curl -LO https://github.com/estesp/manifest-tool/releases/download/v1.0.2/manifest-tool-linux-amd64
+    chmod +x manifest-tool-linux-amd64
+    sudo mv manifest-tool-linux-amd64 /usr/local/bin/manifest-tool
```

Comment:
If this is run locally, we don't want to move stuff into system locations. 

For GH actions, it might be fine, but I think it would still be better to leave the binary in the local dir and call it with `./manifest-tool`.

Reply:
done.